src/main/java/us/dot/its/jpo/ode/aws/depositor/AwsDepositor.java

```diff
@@ -138,17 +142,24 @@ public static void main(String[] args) throws Exception {
 			AWS_SESSION_TOKEN = profile.get(K_AWS_SESSION_TOKEN).toString();
 			AWS_EXPIRATION = profile.get(K_AWS_EXPIRATION).toString().split("\\+")[0];
 		}
-		
 
 		// Properties for the kafka topic
 		Properties props = new Properties();
 		props.put("bootstrap.servers", endpoint);
+
+		props.put("key.deserializer", "org.apache.kafka.common.serialization.StringDeserializer");
```

Comment:
Are these just needed with the new Kafka version?

Reply:
These already existed, I've just moved them a bit higher in the file to match the order of property declarations in other projects.